articles/libraries/lock-android/index.md

```diff
@@ -1,478 +1,248 @@
 ---
+toc_title: Getting Started with Lock for Android
 url: /libraries/lock-android
-description: Basics of Lock for Android
+description: Installation, usage, and configuration options guide for Lock for Android
 ---
 
-::: panel-info Version Warning
-This document is not yet updated to use [Lock for Android](https://github.com/auth0/Lock.Android) 2.0. It will be soon!
-:::
-
 # Lock for Android
 
-[Auth0](https://auth0.com) is an authentication broker that supports social identity providers as well as enterprise identity providers such as Active Directory, LDAP, Google Apps and Salesforce.
-
-## Key features
+Lock for Android can integrate into your native Android apps to provide a beautiful way to log your users in and to sign them up in your app. It provides support for social identity providers such as Facebook, Google, or Twitter, as well as enterprise providers such as Active Directory. You can also use Lock for Android to provide Passwordless authentication using email or SMS. 
 
-* **Integrates** your Android app with **Auth0**.
-* Provides a **beautiful native UI** to log your users in.
-* Provides support for **Social Providers** (Facebook, Twitter, etc.), **Enterprise Providers** (AD, LDAP, etc.) and **Username & Password**.
-* Passwordless authentication using **SMS or Email**.
-
-## Additional Documents
-
-<ul>
-<% _.forEach(_.sortBy(articles.findByHash('libraries/lock-android').items, 'toc_title'), function(article) { %>
-  <% if (article.toc_title) { %>
-  <li>
-    <span><a href="<%- '/docs' + article.url %>"><%- article.toc_title %></a>
-    <% if (article.description) { %>
-      - <%- article.description %>
-    <% } %>
-    </span>
-  </li>
-  <% } %>
-<% }); %>
-</ul>
+Get started using Lock for Android below, or if you're looking for a specific document, try the listing of [additional documents](#additional-documents) related to Lock for Android!!
 
 ## Requirements
 
 Android API level 15+ is required in order to use Lock's UI.
-If you'll create your own API and just call Auth0 API via the `com.auth0.android:core:1.13.+`, the minimum required API level is 9.
+If you intend to create your own API and just call Auth0 API via the `com.auth0.android:auth0:1.1.0`, the minimum required API level is also 15+.
```

Comment:
Also, let's mention https://github.com/auth0/Auth0.Android lib instead (we should probably aim to have a custom UI guide)